functions/supplier/mongodb-supplier/README.adoc

```diff
@@ -28,10 +28,12 @@ For more information on the various options available, please see link:src/main/
 The `queryExpression` and `updateExpression` options may use Spring Data MongoDB query DSL from the `org.springframework.data.mongodb.core.query`, such as `Query` and `Update` factories respectively.
 The `updateExpression` is optional and ca use an item from query result as a root evaluation object to extract some values to update from just fetched data.
 
+A `ComponentCustomizer<MqttPahoMessageHandler>` bean can be added in the target project to provide any custom options for the `MqttPahoMessageHandler` configuration used by the `mqttConsumer`.
```

Comment:
Looks like you got this one and the mqtt function above swapped.

Reply:
I don't know happened to my brain at that moment , but looks like first `m` in their names got me tricked when I had too many opened tabs in the IDE.
Feel free to fix on merge or wait for me in the morning 😅